### PR TITLE
PR-01(presigned_url)

### PR DESCRIPTION
**Pull request by sabbir80**
Fixes #

- [x] Does not depend on other PRs
- [ ] Depends on other PR being merged first: #

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [x] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?
Generate a presigned url using lambda fucntion
## What was done
I have generate a presigned url using lambda fucntion to upload file into s3 bucket
## How to test
This is the url : https://h5ztmr2rhsouhgpvx6wzzfudbe0affou.lambda-url.ap-northeast-1.on.aws/
you have to send queryparameter into filename keyword.e.g filename=something.txt 

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [ ] No risk
- [x] Low
- [ ] High

## Reminder list

- Determine whether REST API Documentation should be updated
- Make an assessment on whether unit tests should be updated and/or added
- Ensure follow up tasks are linked to this PR

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: firstLambda/serverless.yml [hunk 1]
```diff
@@ -4,11 +4,31 @@ frameworkVersion: '3'
...
-    handler: handler.firstLambda
+    handler: handler.lambda_handler
+    url: true
+
+resources:
+  Resources:
+    FileBucket:
+      Type: AWS::S3::Bucket
+      Properties:
+        BucketName: firstlambdabucket-sabbir
```
Comment (mhazad) [inline firstLambda/serverless.yml:32]: This needs to be read from the environment variable.

File: firstLambda/handler.py [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
-
+my_config = Config(
+      signature_version='s3v4',
+    )
 def lambda_handler(event, context):
-    body = {
-        "message": "Go Serverless v3.0! Your function executed successfully!",
-        "input": event,
-    }
+    bucket_name = 'firstlambdabucket-sabbir'
```
[the remark this answers is not in the document]
Comment (sabbir80) [reply firstLambda/handler.py:11]: Okay Bhaia.

File: firstLambda/serverless.yml (hunk 1 above)
Comment (sabbir80) [reply firstLambda/serverless.yml]: addressed the feedback in 0102a45

File: firstLambda/handler.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (sabbir80) [reply firstLambda/handler.py]: addressed the feedback in [0102a45](https://github.com/CraftsmenLtd/coe-batch-q322/pull/355/commits/0102a45f5e2715a243e5a3c1aee67c2a89fc07e4)

**Review by mhazad — APPROVED** [review]
LGTM

**Pull request merged**